content/docs/pipelines/sdk/static-type-checking.md

```diff
@@ -0,0 +1,144 @@
++++
+title = "DSL Static Type Checking"
+description = "Statically check the component I/O types"
+weight = 6
++++
+
+## Overview
+A pipeline is a workflow consisting of [components](/docs/pipelines/sdk/build-component#overview-of-pipelines-and-components) and each
+component contains inputs and outputs. The DSL compiler supports static type checking to ensure the type consistency among the component
+I/Os within the same pipeline. Static type checking helps users to identify component I/O inconsistencies without running the pipeline. 
+It also shortens the development cycles by catching the errors early. 
+This feature is especially useful in two cases: 
+* When the pipeline is huge and manually checking the types is infeasible; 
+* When some components are shared ones and the type information is not immediately available to the pipeline authors.
```

Comment:
Is the "author" a different person from the "user"? I'm thinking they're the same person, and you should change all "author" and "authors" to "you", for clarity. (Similar to my earlier comment about "users".)

When doing these changes, check the surrounding text for other related changes you may need to make, such as changing "their" to "your", etc.

Reply:
SG. The author represents the same group of people, which should be you.

Reply:
Change author to you globally.